internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -797,15 +797,24 @@ func convertPublish(topics []manifest.Topic, accountID, region, app, env, svc st
 	var publishers template.PublishOpts
 	// convert the topics to template Topics
 	for _, topic := range topics {
+		var fifoConfig *template.FIFOTopicConfig
+		if !topic.FIFO.IsEmpty() {
+			fifoConfig = &template.FIFOTopicConfig{
+				Enable: topic.FIFO.Enable,
+			}
```

Comment:
should we keep `fifoConfig` as `nil` if the `topic.FIFO.Enable == false`? Otherwise as far as I can tell we ignore the config:
```yaml
fifo: false
```
And still enable FIFO in CloudFormation.

```go
if topic.FIFO.IsEnabled() {
   fifoConfig = &template.FIFOTopicConfig{}
   if !topic.FIFO.Advanced.IsEmpty() { 
      // Content dedupe stuff...
   }
}
```

Reply:
Makes sense to me.